### ISSUE TITLE
Controls: ControlUnit auto populate issue

### ISSUE THREAD

**Issue by shorty456132** [shorty456132/qsys-vscode-extension#6]
ControlUnit will populate the underlying code. its missing something. bracket or quote?

**Comment by QSC-Eugene** [comment]
the $ before the 1 should be removed
before
```lua
 "body": "ControlUnit = \"${$1|dB,Hz,Float,Integer,Pan,Percent,Position,Seconds|}\","
```
after
```lua
 "body": "ControlUnit = \"${1|dB,Hz,Float,Integer,Pan,Percent,Position,Seconds|}\","
```

**Comment by shorty456132** [comment]
Fixed in 1.2.4
